evergreen/create-expansions.sh

```diff
@@ -19,7 +19,7 @@ if [ "Windows_NT" = "$OS" ]; then
 fi
 export PATH="$GOROOT/bin:$PATH"
 if [[ "${triggered_by_git_tag}" != "" ]]; then
-    export release_version=$(echo ${triggered_by_git_tag} | sed -E 's/^(v|trr|msm|libv)//')
+    export release_version=$(echo ${triggered_by_git_tag} | sed -E 's/^(v|libv)//')
```

Comment:
I randomly noticed that we forgot to remove these two tags.

Reply:
Makes me wish we could automatically generate this sed, but... that's too much.